rcl_lifecycle/CMakeLists.txt

```diff
@@ -11,6 +11,21 @@ find_package(rmw REQUIRED)
 
 include_directories(include)
 
+# Default to C99
+if(NOT CMAKE_C_STANDARD)
+  set(CMAKE_C_STANDARD 99)
+endif()
+
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+
+if(NOT WIN32)
+  set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -Wextra -Wpedantic")
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra -Wpedantic")
```

Comment:
Also done for `rcl_lifecycle` in https://github.com/ros2/rcl/pull/141/commits/cc2c9a5f8ead5708b26fae0f3fa9f56238594339